ui/app/AppLayouts/Wallet/panels/WalletHeader.qml

```diff
@@ -96,7 +96,7 @@ Item {
 
                 font.weight: Font.Normal
                 textColor: Theme.palette.baseColor1
-                text: overview.includeWatchOnly ? qsTr("Hide watch-only"): qsTr("Show watch-only")
+                text: overview.includeWatchOnly ? qsTr("Hide watched address"): qsTr("Show watched address")
```

Comment:
wondering if this one should be "addresses"

Reply:
yes you are right! done :)